sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -301,6 +301,84 @@ extern "C" {
 #ifndef CONFIG_GNRC_SIXLOWPAN_SFR_DG_RETRIES
 #define CONFIG_GNRC_SIXLOWPAN_SFR_DG_RETRIES            0U
 #endif
+
+/**
+ * @brief   The denominator for the factor for when to mark ECN on `netif` input
```

Comment:
```suggestion
 * @brief   The denominator for the factor for when to mark ECN (Explicit Congestion Notification) on `netif` input
```

expanding it once would be enough, those TLAs can be PITA IMHO.

Reply:
Same here, I will add it to the brief of `CONFIG_GNRC_SIXLOWPAN_SFR_USE_ECN`, so it will show up in the first mentioning of ECN in https://doc.riot-os.org/group__net__gnrc__sixlowpan__config.html